jubakit/loader/postgresql.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from ..base import BaseLoader
+from ..compat import *
+from psycopg2 import connect
+from psycopg2.extras import DictCursor
+from psycopg2 import sql
+
+class PostgreSQLoader(BaseLoader):
+  """
+  Loader to process columns of PostgreSQL.
```

Comment:
Would you please add more information about how to use PostgreSQLoader?
The sample code you wrote in the pull-request is good. 
In my opinion, I want to add a `dbname` keyword argument in the sample code. 

```
auth = PostgreSQLAuthHandler(user='postgres', password='postgres', host='localhost', port='5432')

loader = PostgreSQLoader(auth, table='test')
for row in loader:
  print(row)

# [1, 100, 'abcdef']
# [2, 200, 'ghijkl']
# [3, 300, 'mnopqr']
```